Source/WebCore/page/ContextMenuController.cpp

```diff
@@ -374,6 +374,11 @@ void ContextMenuController::contextMenuItemSelected(ContextMenuAction action, co
     case ContextMenuItemTagToggleVideoEnhancedFullscreen:
         m_context.hitTestResult().toggleEnhancedFullscreenForVideo();
         break;
+#if PLATFORM(MAC)
```

Comment:
Is the `#if PLATFORM(MAC)` needed here? The code it's guarding doesn't seem Mac-specific.

Reply:
Fixed!